docusaurus/docs/iOS/client/llc-client-overview.md

```diff
@@ -9,61 +9,14 @@ Both our [UIKit](../uikit/getting-started.md) and [SwiftUI](../swiftui/getting-s
 
 ## Accessing State with Controllers
 
-The `StreamChat` framework comes with controllers and delegates that you can use to build your own views.
+The `StreamChat` framework comes with [controllers and delegates](controllers/controllers-overview.md) that you can use to build your own views.
 
 Each controller exposes API functionality and supports delegation. Controllers and their delegates are documented here based on the kind of data they control and allow you to observe. You can find examples on how to build your own view as well.
 
-In addition to controllers, we provide a modern [state layer](state-layer/state-layer-overview.md) with async-await and observable state objects.
+## Accessing State with State Layer
 
-## Using Controllers
+In addition to controllers, we provide a modern [state layer](state-layer/state-layer-overview.md). It follows an architecture where we have objects interacting with the Stream API through async functions. These objects are accompanied with state objects that hold the current state.
```

Comment:
The only thing weird here is that we talk about the controllers, then the state layer, and then back to the controllers 🤔 

Ideally, we would talk first about the State Layer and saying it is the new way of managing state, and only then we talk about the controllers

Reply:
Changed it a bit.